mail_tracking_mailgun/models/mail_tracking_email.py

```diff
@@ -240,9 +240,9 @@ def action_manual_check_mailgun(self):
                 raise ValidationError(_("Event information not longer stored"))
             for item in content["items"]:
                 if not self.env['mail.tracking.event'].search(
-                        [('mailgun_id', '=', item["id"])]):
+                        [('mailgun_id', '=', item.get("id"))]):
```

Comment:
You can't enter here if there's no ID, but I don't know why this can happen

Reply:
Well, if there's an ok response there should allways be an id.

Reply:
But again, if there's no ID, you shouldn't perform any search IMO.